tools/c7n_left/c7n_left/cli.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+from pathlib import Path
+
+import click
+from c7n.config import Config
+
+from .output import get_reporter, report_outputs
+from .provider import CollectionRunner
+from .utils import load_policies
+
+
+@click.group()
+def cli():
+    """Shift Left Policy"""
+    logging.basicConfig(level=logging.DEBUG)
+
+
+@cli.command()
+@click.option("--format", default="terraform")
+@click.option("-p", "--policy-dir", type=click.Path())
```

Comment:
This might benefit from the ability to specify patterns or individual files, though it's also handy to cover a full directory :+1:

Reply:
intent is to add include/exclude policy syntax in future rev.

Reply:
Makes sense :+1: